src/main/java/com/palechip/hudpixelmod/chat/WarlordsDamageChatDisable.java

```diff
@@ -0,0 +1,28 @@
+package com.palechip.hudpixelmod.chat;
+
+import com.palechip.hudpixelmod.HudPixelConfig;
+
+import net.minecraftforge.client.event.ClientChatReceivedEvent;
+
+public class WarlordsDamageChatDisable {
+    public static final String takeDamage = "�";
+    public static final String giveDamage = "�";
```

Comment:
Please please test a build of the mod. I'm fairly certain that Java will screw up the `»` an the `«` causing the feature not to work. A similar thing happened to me with the color code sign in the past. You'll probably have to write it using \u...